gubernator/update_config.sh

```diff
@@ -16,5 +16,5 @@
 # This script updates the Gubernator configuration
 # file to keep it in sync with Prow.
 
-cd "$( dirname "${BASH_SOURCE[0]}" )"
-./update_config.py ./../prow/config.yaml ./config.yaml
\ No newline at end of file
+cd "$( dirname "${BASH_SOURCE[0]}" )" || exit 1
```

Comment:
Why not just `set -o errexit`?

Reply:
bumping this. imo `set -o errexit` is a better idea in case anyone adds to this later.